docs/_deploying/heroku.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Deploying to Heroku
+order: 0
+slug: heroku
+lang: en
+layout: tutorial
+permalink: /deploying/heroku
+redirect_from:
+  - /deploying
+---
+# Deploying to Heroku
+
+<div class="section-content">
+This guide will walk you through preparing and deploying a Slack app using [Bolt for JavaScript](/bolt-js) and the [Heroku platform](https://heroku.com/). Along the way, we’ll download a Bolt Slack app, prepare it for Heroku, and deploy it.
+</div>
+
+When you’re finished, you’ll have this ⚡️[Getting Started with Heroku app](https://github.com/mwbrooks/bolt-js-getting-started-with-heroku-app) to run, modify, and make your own.
+
+---
+
+### Get a Bolt Slack app
+
+We'll continue from [Getting Started with Bolt for JavaScript][getting-started-guide] to learn how to prepare and deploy the Getting Started app to Heroku.
+
+Download the Getting Started app and change into its directory:
+
+```shell
+git clone https://github.com/mwbrooks/bolt-js-getting-started-app.git
+cd bolt-js-getting-started-app/
+```
```

Comment:
Question: every command after this section assumes that you're inside the app directory from running `cd bolt-js-getting-started-app/`. However, some users may be reading along with their own app and may not run this command. That could cause issues later on in the guide.

An especially problematic issue would be initializing a git repository in a directory that's not the app directory.

Do you think it's worth adding a note that we assume every command from here on is run inside the app directory? This wasn't called out in the _Getting Started_ guide, which is why I didn't in this guide.

Reply:
I think most developers don't have the issue but if you could have such a kind guide here, it'll be much more beginner-friendly.

Reply:
Based on @shaydewael's feedback, we separated out these two commands. This makes it easier to follow for developers who are using an existing Bolt app and clarifies the importance of changing into the app's directory.